app/containers/HomePage/index.js

```diff
@@ -1,24 +1,83 @@
-/*
- * HomePage
+/**
  *
- * This is the first thing users see of our App, at the '/' route
+ * HomePage
  *
- * NOTE: while this component should technically be a stateless functional
- * component (SFC), hot reloading does not currently support SFCs. If hot
- * reloading is not a necessity for you then you can refactor it and remove
- * the linting exception.
  */
 
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
-import messages from './messages';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Helmet } from 'react-helmet';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { push } from 'react-router-redux';
+
+import styled from 'styled-components';
+import BigBtn1 from 'components/BigBtn1';
+import { getActive } from 'themes';
+import PageHeader from 'components/PageHeader/index';
+
+import injectSaga from 'utils/injectSaga';
+import injectReducer from 'utils/injectReducer';
+import makeSelectHomePage from './selectors';
+import reducer from './reducer';
+import saga from './saga';
 
-export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+export class HomePage extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.navigate = this.navigate.bind(this);
+  }
+  navigate(path) {
+    this.props.dispatch(push(path));
+  }
   render() {
+    const theme = getActive();
+    const FlexBox = styled.div`
+      display: -webkit-flex;
+      display: flex;
+      -webkit-flex-direction: column;
+      flex-direction: column;
+    `;
     return (
-      <h1>
-        <FormattedMessage {...messages.header} />
-      </h1>
+      <div>
+        <Helmet>
+          <title>Home</title>
+          <meta name="description" content="Home Page" />
+        </Helmet>
+        <PageHeader title="My Chemical Bromance" />
+        <br />
+        <FlexBox >
+          <BigBtn1 tab="1%" theme={theme} text="View Database" link="/database" handler={(link) => this.navigate(link)} />
```

Comment:
If we are linking in this manner then it is especially important that we write test cases to verify these links so we don't forget about them when we make changes to their respective pages.